parsers/CA_YT.py

```diff
@@ -13,7 +13,7 @@
 
 def fetch_production(
     zone_key: str = "CA-YT",
-    session: Optional[Session] = None,
+    session: Session = Session(),
```

Comment:
Btw this is a python quirk, but it can actually lead to some weird problems if you use mutable objects as default parameters: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/